### PR TITLE
Fix #699

### LINKED ISSUE #699: Wrong config reading in cflow?
https://github.com/openvinotoolkit/anomalib/blob/15625d6dd09b5dcf043aa4f4969c1c968173fe6e/anomalib/models/cflow/lightning_model.py#L189

Shouldn't it be `fiber_batch_size=hparams.model.fiber_batch_size` ?
 
Instead of `fiber_batch_size=hparams.dataset.fiber_batch_size`. All other parameters are from `hparams.model`.

---

There is also this inconsistent `permute_soft=hparams.model.soft_permutation`.
I guess it should be `permute_soft=hparams.model.permute_soft`, no?

### PR DESCRIPTION
**Pull request by jpcbertoldo**
- Fixes #699

## Changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which refactors the code base)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [] My code follows the [pre-commit style and check guidelines](https://github.com/openvinotoolkit/anomalib/blob/main/CONTRIBUTING.md) of this project.
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: anomalib/models/cflow/config.yaml [hunk 1]
```diff
@@ -27,6 +27,7 @@ model:
   condition_vector: 128
   coupling_blocks: 8
   clamp_alpha: 1.9
+  fiber_batch_size: 64
   soft_permutation: false
```
Comment (ashwinvaidya17) [inline anomalib/models/cflow/config.yaml:31]: Shouldn't this be renamed to `permute_soft`?

File: anomalib/models/cflow/config.yaml [hunk 2]
```diff
@@ -27,6 +27,7 @@ model:
   condition_vector: 128
   coupling_blocks: 8
   clamp_alpha: 1.9
+  fiber_batch_size: 64
```
Comment (ashwinvaidya17) [inline anomalib/models/cflow/config.yaml:30]: Since it is here now we can remove this from datasets.

**Review by ashwinvaidya17 — CHANGES_REQUESTED** [review]
I think the updated config is not synced with github.

**Comment by samet-akcay** [comment]
I have the same comments

**Comment by jpcbertoldo** [comment]
> I think the updated config is not synced with github.

I'm not sure what you mean but i changed what you said in the line comments.

**Comment by ashwinvaidya17** [comment]
> > I think the updated config is not synced with github.
> 
> I'm not sure what you mean but i changed what you said in the line comments.

I meant maybe you made the changes locally but forgot to push them.

**Comment by jpcbertoldo** [comment]
The error is not the usual coverage upload but I'm not sure it is related to the PR.

image

**Comment by samet-akcay** [comment]
> The error is not the usual coverage upload but I'm not sure it is related to the PR.
> 
> image

Based on the [logs](https://github.com/openvinotoolkit/anomalib/actions/runs/3420984937/jobs/5696632200), looks like there is something wrong with this tests.
`tests/pre_merge/utils/callbacks/normalization_callback/test_normalization_callback.py F [ 95%]`

**Comment by jpcbertoldo** [comment]
> > The error is not the usual coverage upload but I'm not sure it is related to the PR.
> > image
> 
> Based on the [logs](https://github.com/openvinotoolkit/anomalib/actions/runs/3420984937/jobs/5696632200), looks like there is something wrong with this tests. `tests/pre_merge/utils/callbacks/normalization_callback/test_normalization_callback.py F [ 95%]`

Thanks!
Will check it locally.

**Comment by jpcbertoldo** [comment]
```
pre_merge_pytest run-test: commands[0] | python -m pytest /cluster/CMM/home/jcasagrandebertoldo/repos/anomalib/tests/pre_merge/utils/callbacks/normalization_callback/test_normalization_callback.py --showlocals
==================================================================================== test session starts ====================================================================================
platform linux -- Python 3.8.13, pytest-7.2.0, pluggy-1.0.0
cachedir: .tox/pre_merge_pytest/.pytest_cache
rootdir: /cluster/CMM/home/jcasagrandebertoldo/repos/anomalib
plugins: flaky-3.7.0, anyio-3.6.2
collected 1 item                                                                                                                                                                            

tests/pre_merge/utils/callbacks/normalization_callback/test_normalization_callback.py .                                                                                               [100%]

======================================================================== 1 passed, 33 warnings in 123.98s (0:02:03) =========================================================================
__________________________________________________________________________________________ summary __________________________________________________________________________________________
  pre_merge_pytest: commands succeeded
  congratulations :)
 *  Terminal will be reused by tasks, press any key to close it. 
 ```
 
No problems in my local tox :thinking:

**Comment by jpcbertoldo** [comment]
Maybe some flaky effect? Is there a way to relaunch it @samet-akcay ?

**Comment by samet-akcay** [comment]
> Maybe some flaky effect? Is there a way to relaunch it @samet-akcay ?

Re-running the tests now...

**Comment by jpcbertoldo** [comment]
test_normalization_callback

> > Maybe some flaky effect? Is there a way to relaunch it @samet-akcay ?
> 
> Re-running the tests now...

Yeah, still.

I'm clueless here atm. I tried to search for "cflow" in the logs and there is nothing :thinking:

**Comment by samet-akcay** [comment]
> test_normalization_callback
> 
> > > Maybe some flaky effect? Is there a way to relaunch it @samet-akcay ?
> > 
> > 
> > Re-running the tests now...
> 
> Yeah, still.
> 
> I'm clueless here atm. I tried to search for "cflow" in the logs and there is nothing 🤔
> 
> Rebasing to see what happens.

Looks like there is something wrong with the tests. I'm observing a similar behaviour on another PR as well. Something seems to be broken.

**Comment by jpcbertoldo** [comment]
> > test_normalization_callback
> > > > Maybe some flaky effect? Is there a way to relaunch it @samet-akcay ?
> > > 
> > > 
> > > Re-running the tests now...
> > 
> > 
> > Yeah, still.
> > I'm clueless here atm. I tried to search for "cflow" in the logs and there is nothing thinking
> > Rebasing to see what happens.
> 
> Looks like there is something wrong with the tests. I'm observing a similar behaviour on another PR as well. Something seems to be broken.

There are also some badly configured tests I could not pass locally but they pass in the CI apparently.
See #704.
But it's another test: `tests/pre_merge/utils/callbacks/visualizer_callback/test_visualizer.py`

**Comment by jpcbertoldo** [comment]
I think it's mergeable now?
The error now is the usual coverage report.

**Comment by samet-akcay** [comment]
> I think it's mergeable now? The error now is the usual coverage report.

Yes, let's merge it! Thanks for the effort!

**Pull request merged**